Marlin/ultralcd.cpp

```diff
@@ -57,6 +57,10 @@
   extern void mesh_probing_done();
 #endif
 
+#if ENABLED(AUTO_BED_LEVELING_UBL) ||  ENABLED(UBL_G26_MESH_VALIDATION)
+  bool has_control_of_lcd_panel;
```

Comment:
Since the name was predicated on having `ubl.` in front of it, a better new name for this flag might be `lcd_under_external_control`.

Reply:
Agreed...